main-app/app/scripts/bingoTicket/bingoModel.js

```diff
@@ -6,32 +6,47 @@
             me.tickets = [];
             me.bingoStrip = [];
 
-            var clearOldTickets = function () {
-                me.tickets = [];
-                me.bingoStrip = [];
-            };
-
             me.getTicket = function (ticketFromServer) {
                 clearOldTickets();
                 generateTickets(ticketFromServer);
-                for (var i = 0; i < me.tickets.length; i++) {
+                for(var i = 0; i < me.tickets.length; i++){
                     createBingoTicketGrid();
                     splitingNumbersUp(me.tickets[i]);
                 }
             };
 
+            var clearOldTickets = function () {
+                me.tickets = [];
+                me.bingoStrip = [];
+            };
+
+            var BingoCell = function () {
+                this.bingoNumber = null;
+                this.cssClass = "unmarked";
+                this.counted = false;
+            };
+
+            var TicketInfo = function () {
+                this.row1Marked = 0;
+                this.row2Marked = 0;
+                this.row3Marked = 0;
+                this.numbersToWin = 5;
+            };
+
             var createBingoTicketGrid = function () {
                 me.ticketGrid = [
-                    [null, null, null, null, null, null, null, null, null],
-                    [null, null, null, null, null, null, null, null, null],
-                    [null, null, null, null, null, null, null, null, null]
+                    [new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell()],
+                    [new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell()],
+                    [new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell(), new BingoCell()],
+                    [new TicketInfo()]
                 ];
             };
 
+
             var generateTickets = function (ticket) {
```

Comment:
You'll have to work through the consequences of the above stuff on the remaining code:

Splitting into an array  & sorting is the work of other object(s) (if you think about a pure ticket - it also breaks the SRP)

Checking row for numbers becomes easier with the changes to `BingoCell` and the new `TicketLine`